lighthouse-cli/test/smokehouse/smoke-test-dfns.js

```diff
@@ -122,6 +128,7 @@ function getSmokeTests() {
       id: smokeTestDfn.id,
       config: loadConfig(resolveLocalOrProjectRoot(smokeTestDfn.config)),
       expectations: loadExpectations(resolveLocalOrProjectRoot(smokeTestDfn.expectations)),
+      flagsPath: smokeTestDfn.flags,
```

Comment:
the fact that this is differently named makes it a bit confusing, standardize on `flagsPath`? it's just a mistake that `config` isn't `configPath` IMO anyway :)

Reply:
change all these to `*Path`?